### DISCUSSION TITLE
Some thoughts and suggestions.

### DISCUSSION THREAD

**Discussion by devanteweary** [bitmagnet-io/bitmagnet#307] [Ideas]
Well I like the concept but this thing has created nearly 4GB in less than 24 hours so there's no way using this is sustainable.

Another thing that happened was as soon as I connected it to prowlarr and let it run for like 30min, a BUNCH of movies started upgrading.  

Which is cool I guess but several of them didn't follow the custom profile I have set up. 

For instance, I have negative score for Hindi language in my movies so that I never get Hindi, but the process of using bitmagnet completely ignored that.

Several movie upgrades just flat out ignored my setup like that.  

And one said it was one movie, then when I go to watch it, was a completely different movie (probably nothing to do with bitmagnet but also has never happened before).

I want to like this app but those two problems mean I have to turn it off.

And if I can make suggestions, of course dark mode but also exclusions. 

Like I don't care about adult torrents, music, comics. So it'd be nice to disable those.

Alright that's all and like I said, great concept!

**Comment by RedSunEmpire** [comment]
> but also exclusions. 
> Like I don't care about adult torrents, music, comics. So it'd be nice to disable those.

You can already do that via classifier settings: https://bitmagnet.io/guides/classifier.html#auto-delete-specific-content-types

I'll let more knowledged people answer your other points since i'm also new to the project.

Also, please keep in mind this software is still in _**alpha**_ :)

**Comment by DyonR** [comment]
> nearly 4GB in less than 24 hours 

My bitmagnet instance has ~18 million torrents, and is 160GB. In the beginning it goes rapid, but as it indexes more torrents, there is less to discover, thus recuding the "GB/day".  
Mine has been running for 3 months.  
![afbeelding](https://github.com/user-attachments/assets/9a15e801-e1f2-4506-8d25-4aaf57fde463)
  
Edit 2024-11-08: My `dht_crawler.save_files_threshold` is set to `500`, the default is `100`, and my `dht_crawler.scaling_factor` is set to `50`, the default is `10`.

**Comment by binhex** [comment]
Looks like @devanteweary  hoped over to this repo hoping the experience would be different to my image :-) https://forums.unraid.net/topic/174999-support-binhex-bitmagnet/?do=findComment&comment=1465052 or maybe he was running both at the same time, i dunno, either way complaining that bitmagnet is finding too many magnets is novel feedback :-).

FWIW i have included an[ example config.yml](https://github.com/binhex/arch-bitmagnet/blob/d40e4c2e8169b8053ecf259a4461d391c2ab4f2e/config/nobody/config.yml.example) in my image which will help reduce the number of magnets stored, feel free to re-use it if it's of any use to you.

**Reply by devanteweary** [reply]
> Looks like @devanteweary hoped over to this repo hoping the experience would be different to my image :-) [forums.unraid.net/topic/174999-support-binhex-bitmagnet?do=findComment&comment=1465052](https://forums.unraid.net/topic/174999-support-binhex-bitmagnet/?do=findComment&comment=1465052) or maybe he was running both at the same time, i dunno, either way complaining that bitmagnet is finding too many magnets is novel feedback :-).
> 
> FWIW i have included an[ example config.yml](https://github.com/binhex/arch-bitmagnet/blob/d40e4c2e8169b8053ecf259a4461d391c2ab4f2e/config/nobody/config.yml.example) in my image which will help reduce the number of magnets stored, feel free to re-use it if it's of any use to you.

Well firstly I don't really follow who makes the different containers or even the differences between them. 
Heck, without checking right now, I don't even know which "version" I got.
(be nice if there was a guide on what binhex vs. hotio means, for example).

So I didn't know this one was different.

Secondly, my complaint as you call it is more like personal commentary.  
I personally can't handle all that space being used just to list torrents. Others can.
Obviously not complaining about the number of magnets. Just the space it uses.

And lastly, part of the reason for making a discussion is to hear others' opinion and possibly tips.  
For instance, @DyonR says his is kinda levelling around 160GB... well that I can handle so now that I know that might be a possible level where it might level out, I'll probably let it ride. And you gave me a sample config so even better.

Thanks for the config, I'll definitely check it out. :>

**Comment by flowcool** [comment]
-> For instance, I have negative score for Hindi language in my movies so that I never get Hindi, but the process of using bitmagnet completely ignored that.

How did you manage to do this ?

For my case, I'd like to keep only french but struggle to find how. If anyone have an idea...

**Reply by devanteweary** [reply]
> -> For instance, I have negative score for Hindi language in my movies so that I never get Hindi, but the process of using bitmagnet completely ignored that.
> 
> How did you manage to do this ?
> 
> For my case, I'd like to keep only french but struggle to find how. If anyone have an idea...

Not sure if it's the BEST way, but here's how I do it.

1) Radarr > Settings > Custom Format > Create one per desired and undesired language.
2) Radarr > Settings > Profiles > Select your Quality Profile > Minimum Custom Score 1.
3) Radarr > Settings > Profiles > Custom Formats > Give negative score to undesired language.

Here are my screenshots.  
The idea is I prefer English, but will take and don't want to exclude JP, KO, CH, RU, SP.  

Ultimately I want it to upgrade to English and x265.
All those 5 points means even if a torrent had several of those AND was x265, it still wouldn't hit the desired 50 points. 
Theoretically if something was English and like, 4 other languages, it would hit the 50 (thereby excluding x265) but I haven't really had that happen.

The reason for -100/-200 is just so I can tell at a glance why a torrent might have a negative score (for language or for video type). They could easily be all -100. As long as the score makes it 0 or below.

![image](https://github.com/user-attachments/assets/1700d2e2-63bb-4537-b00e-6797db2c1775)

![image](https://github.com/user-attachments/assets/25c443eb-d743-4968-af98-94e316bb4685)
